fcrepo-auth-common/src/test/java/org/fcrepo/auth/integration/AbstractResourceIT.java

```diff
@@ -48,8 +52,8 @@ public void setLogger() {
         logger = LoggerFactory.getLogger(this.getClass());
     }
 
-    private static final int SERVER_PORT = Integer.parseInt(System
-            .getProperty("fcrepo.dynamic.test.port", "8080"));
+    private static final int SERVER_PORT = parseInt(Objects.requireNonNullElse(
+            Strings.emptyToNull(System.getProperty("fcrepo.dynamic.test.port")), "8080"));
```

Comment:
What purpose does looking for `fcrepo.dynamic.test.port` service, since you removed its being set in the pom.xml? https://github.com/fcrepo4/fcrepo4/pull/1670/files#diff-44094b67d49071d47bb32efe43678722L70

Reply:
?

The property is set by the build-maven-helper-plugin when it reserves a port. The line I removed was the line that @bbpennel added that original broke the dynamic port assignment.